### PR TITLE
Added "Setup Assistant" example

### PR DESCRIPTION
**Pull request by nmcspadden**
A more complex example using window-loading functions, automatic
running of code, and timers.

### CODE REVIEW

File: examples/setup_assistant.py [hunk 1]
```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/python
+"""Nibbler."""
```
Comment (clburlison) [inline examples/setup_assistant.py]: I'd remove the doc string or update it to setup assistant

File: examples/setup_assistant.py [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+
+import argparse
+import logging
+import os
+
+# black magic
+MapKit = objc.loadBundle(
+  'MapKit',
+  globals(), bundle_path='/System/Library/Frameworks/MapKit.framework'
+)
```
Comment (clburlison) [inline examples/setup_assistant.py]: You're not actually using MapKit. You should remove it.

**Review by clburlison — COMMENTED** [review]
Pretty cosmetic changes.

File: examples/setup_assistant.py (hunk 2 above)
Comment (nmcspadden) [reply examples/setup_assistant.py]: It's in the sam_example.py script, but doesn't seem to be used there. Wasn't sure if it actually did some 'black magic' as the comment in sam_example.py implies.

Comment (clburlison) [reply examples/setup_assistant.py]: For reference purposes it's because the `sam.nib` file contains a Map.

**Comment by nmcspadden** [comment]
ping

**Pull request merged**